src/status_im/contexts/profile/profiles/view.cljs

```diff
@@ -183,22 +182,21 @@
        :render-fn               profile-card}]]))
 
 (defn password-input
-  [processing error]
-  (let [auth-method         (rf/sub [:auth-method])
```

Comment:
Hi @clauxx, thanks for noticing.

The behaviour is a bit different, this subscription has been replaced by another one that calculates the `auth-method` for all the profiles instead of calculating it for the pressed profile. this change let us have a better UX when the user switches profiles.

It comes from the parent because there the `auth-method` is stored in the profile.

LMK if you have another question!